src/server.c

```diff
@@ -2633,6 +2633,7 @@ void initServer(void) {
     server.blocking_op_nesting = 0;
     server.thp_enabled = 0;
     server.cluster_drop_packet_filter = -1;
+    server.dbg_node_send_extensions = 1;
```

Comment:
```suggestion
    server.dbg_suppress_clustermsg_extensions = 0;
```

Reply:
also, I think we should move all dbg_* settings into its own "sections" and not mixed normal server settings